File: app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsSpeakerView.java
package net.squanchy.eventdetails.widget;

import android.content.Context;
import android.util.AttributeSet;
import android.view.ViewGroup;
import android.widget.ImageView;

import net.squanchy.R;
import net.squanchy.support.widget.SpeakerView;

public class EventDetailsSpeakerView extends SpeakerView {

    public EventDetailsSpeakerView(Context context, AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public EventDetailsSpeakerView(Context context, AttributeSet attrs, int defStyleAttr) {
        this(context, attrs, defStyleAttr, 0);
    }

    public EventDetailsSpeakerView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }

    @Override
    protected ImageView inflatePhotoView(ViewGroup speakerPhotoContainer) {
        return (ImageView) layoutInflater().inflate(R.layout.view_speaker_photo_event_details, speakerPhotoContainer, false);
    }
}

File: app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsSpeakerView.kt
package net.squanchy.eventdetails.widget

import android.content.Context
import android.util.AttributeSet
import android.view.ViewGroup
import android.widget.ImageView

import net.squanchy.R
import net.squanchy.support.widget.SpeakerView

class EventDetailsSpeakerView @JvmOverloads constructor(
        context: Context,
        attrs: AttributeSet,
        defStyleAttr: Int = 0,
        defStyleRes: Int = 0
) : SpeakerView(context, attrs, defStyleAttr, defStyleRes) {

    override fun inflatePhotoView(speakerPhotoContainer: ViewGroup): ImageView {
        return layoutInflater().inflate(R.layout.view_speaker_photo_event_details, speakerPhotoContainer, false) as ImageView
    }
}
